src/main/java/jp/ac/keio/bio/fun/xitosbml/CuiMainImgSpatial.java

```diff
@@ -0,0 +1,78 @@
+package jp.ac.keio.bio.fun.xitosbml;
+
+import java.io.File;
+import java.io.IOException;
+
+import javax.xml.stream.XMLStreamException;
+
+import org.sbml.jsbml.SBMLException;
+import org.sbml.jsbml.SBMLWriter;
+import org.sbml.jsbml.TidySBMLWriter;
+
+import ij.IJ;
+import ij.ImagePlus;
+import jp.ac.keio.bio.fun.xitosbml.util.ModelValidator;
+import jp.ac.keio.bio.fun.xitosbml.xitosbml.SpatialSBMLExporter;
+
+public class CuiMainImgSpatial extends CuiMainSpatial {
+
+	protected ImagePlus imager;
+
+	public void run(String arg)
+	{
+		//empty
+	}
+	
+	public void runCui(String path1, String path2) {
+		ImagePlus imager = new ImagePlus(path1);
+		this.imager = imager;
+		cui(imager);
+		computeImgTrial();
+		SpatialSBMLExporter sbmlexp = new SpatialSBMLExporter(spImg);
+		model = sbmlexp.getModel();
+		sbmlexp.createGeometryElements(); // visualize(spImg);
+
+		sbmlexp.addCoordParameter();
+		document = sbmlexp.getDocument();
+
+		//To save the SBML document
+		try {
+			SBMLWriter.write(document, new File(path2), ' ', (short) 2);
+			System.out.println("Saved model");
+		} catch (NullPointerException e) {
+			System.out.println("SBML document was not saved");
+		} catch (SBMLException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		} catch (XMLStreamException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+
+		try {
+			String docStr = new TidySBMLWriter().writeSBMLToString(document);
+			IJ.log(docStr);
```

Comment:
I haven't checked yet on my side but, is `IJ.log` printed out in stdout or does a log window show up?

Reply:
No log window opens up; the IJ.log is printed in the stdout.